vncviewer/Viewport.cxx

```diff
@@ -589,6 +590,7 @@ int Viewport::handle(int event)
 
   case FL_LEAVE:
     window()->cursor(FL_CURSOR_DEFAULT);
+    mouseButtonMask = 0;
```

Comment:
Have you checked for cases where the system might steal things for us? E.g. Alt+Tab? Or screenshots?

Do we get a `FL_LEAVE` in those cases? Or do we need to check for something else?

Reply:
https://github.com/TigerVNC/tigervnc/pull/1826#discussion_r1773230996